robin_stringer/app.js

```diff
@@ -0,0 +1,35 @@
+
+'use strict';
+
+var express = require('express');
+var app = express();
+var fs = require('fs');
+var bodyParser = require('body-parser');
+var http = require('http');
+
+//with thanks to Treehouse for tutorial on bodyParser
+//(https://teamtreehouse.com/library/express-basics)
+app.use(bodyParser.json());
+
+app.get('/answers', function(req, res, next){
+  fs.readFile(__dirname + '/../data/answers.json', function(err, data){
+    if(err) throw err;
+    next();
+  });
+  res.send(answers.toString());
+});
+
+app.post('/answers', function(req, res, next) {
+  req.on('data', function(data) {
+    var dataLogged = data.toString();
+    fs.writeFile(__dirname + '/../data/answers.json', dataLogged, function() {
+    if(err) throw err;
+    next();
+    });
+  });
+  res.send('POST request to the homepage');
+});
```

Comment:
some folks generalized the requests a little more using parameters. They used /:item as the route and then used req.params.item in naming the file.